.github/workflows/ci_workflows.yml

```diff
@@ -72,6 +72,12 @@ jobs:
             toxposargs: --remote-data --run-slow
             allow_failure: true
 
+          - name: Python 3.11 with stable versions of dependencies and Roman
+            python: '3.11'
+            runs-on: ubuntu-latest
+            toxenv: py311-test-romandeps
+            allow_failure: true
```

Comment:
@pllim Is this the correct transmutation of the [previous cron version](https://github.com/spacetelescope/jdaviz/pull/3353/files#diff-a35c80c05722587c6753fbb714da8527c91b56ed832a898f58b18633dd5af10aL57)?

Reply:
There was a mistake. I pushed commit to hopefully fix it. Does Roman pull in remote data? I think it does, right?